__tests__/app.test.js

```diff
@@ -10,64 +10,78 @@ afterAll(() => db.end());
-      return request(app)
-          .get('/api')
-          .expect(200)
-          .then(({body}) => {
-              expect(body).toEqual(endpoints)
-          })        
+    return request(app)
+      .get('/api')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(endpoints)
+      })
   })
 })
 
 describe('Test GET api/topics  endpoint', () => {
-    test('Test connection with GET api/topics, should receive 200 as the status code to show connection', () => {
-        return request(app)
-            .get('/api/topics')
-            .expect(200)                  
-    })
-    test('Test connection with GET api/topics, should receive all topics which should have keys (slug and description)', () => {
-        return request(app)
-            .get('/api/topics')
-            .expect(200)
-            .then(({body}) => {
-                expect(body.length > 0).toBe(true)
-                expect(body).toEqual([{"description": "The man, the Mitch, the legend", "slug": "mitch"}, 
-                  {"description": "Not dogs", "slug": "cats"}, 
-                  {"description": "what books are made of", "slug": "paper"}
-                ])
-            })        
-    })
+  test('Test connection with GET api/topics, should receive 200 as the status code to show connection', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+  })
+  test('Test connection with GET api/topics, should receive all topics which should have keys (slug and description)', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length > 0).toBe(true)
+        expect(body).toEqual([{ "description": "The man, the Mitch, the legend", "slug": "mitch" },
+        { "description": "Not dogs", "slug": "cats" },
+        { "description": "what books are made of", "slug": "paper" }
+        ])
+      })
+  })
 })
 
 describe('Test GET api/articles  endpoint', () => {
   test('Test connection with GET api/articles, should receive 200 as the status code to show connection', () => {
-      return request(app)
-          .get('/api/articles/1')
-          .expect(200)                  
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+  })
+  test('Test connection with GET api/article, should receive an article', () => {
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({Article : {
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        }})
+      })
+  })
```

Comment:
Good to see that you've taken on board the feedback to send this back on a key of 'Articles', however I can't see any equivalent controller that is handling this functionality on the response for this endpoint. You will need a separate controller and model for getting article by id to separate the concerns. This controller will need to send back {article: article} in order to satisfy the test.